autoload/autotag.vim

```diff
@@ -0,0 +1,15 @@
+if has("python3")
```

Comment:
This can also be:

``` vim
if !has('python3')
  finish
endif
```

Which would remove one level of indentation from the file, same for the plugin file.

Reply:
I pushed that, thanks.